examples/README.md

```diff
@@ -0,0 +1,43 @@
+# Rapid Java SDK Examples
+
+This repository contains examples of how to use the Expedia Group Rapid Java SDK. The examples are written in Java and use Maven for dependency management.
+
+## Examples
+
+The example implementation provided in the examples package demonstrate different scenarios utilizing various RAPID APIs through the SDK:
+
+The scenarios below specifically demonstrate the usage of booking APIs:
+
+1.`SingleRoomBookScenario.java`: This example demonstrates the process of booking a single room. It includes steps like getting property availability, checking room prices, and finally booking a room.
```

Comment:
We could add _relative_ links to all the classes mentioned in the file (as well as the "examples" project).